src/slic3r/GUI/Mouse3DController.cpp

```diff
@@ -43,14 +43,15 @@ static const std::vector<int> _3DCONNEXION_DEVICES =
     0xc628,	/* 50728 space navigator for notebooks*/
     0xc629,	/* 50729 space pilot pro*/
     0xc62b,	/* 50731 space mouse pro*/
+    0xc640,	/* 50752 nulooq */
```

Comment:
Any reason to move it here? It looks like the list is sorted otherwise.

Reply:
so it's sorted in the same order as the upstream list